packages/ses/README.md

```diff
@@ -137,6 +146,61 @@ c1.globalThis === c2.globalThis; // false
 c1.globalThis.JSON === c2.globalThis.JSON; // true
 ```
 
+The global scope of every compartment includes a shallow, specialized copy of
+the JavaScript intrinsics, omitting `Date.now` and `Math.random` since these
+can be used as covert communication channels between programs that have not
+been expressly introduced.
```

Comment:
Long again. How about "Every compartment's global scope includes a shallow, specialized, copy of the JavaScript intrinsics, except for `Date.now` and `Math.random`. These are left out since they can be used as covert communication channels between programs that have not been expressly introduced."